src/commands/auth.rs

```diff
@@ -5,22 +5,150 @@ use api::ApiClient;
 use colored::Colorize;
 use credentials::CredentialsStorage;
 use models::ResultWithDefaultError;
+use std::io::Write;
 
 pub struct AuthenticationCommand;
 
+const AUTH_SUCCEEDED_MESSAGE: &str = "Successfully authenticated for user with email:";
+
 impl AuthenticationCommand {
-    pub async fn execute(
+    pub async fn execute<W: Write>(
+        mut writer: W,
         api_client: impl ApiClient,
         credentials_storage: impl CredentialsStorage,
     ) -> ResultWithDefaultError<()> {
         let user = api_client.get_user().await?;
         credentials_storage.persist(user.api_token)?;
-        println!(
+        write!(
```

Comment:
```suggestion
        writeln!(
```

Since we're outputting to the `stdout` directly, we need to insert the newline character ourselves, thankfully rust has a `writeln` macro that does exactly that.